### ISSUE TITLE
Indicator for hidden comments

### ISSUE THREAD

**Issue by Nutomic** [LemmyNet/lemmy-ui#999]
### Is your proposal related to a problem?

Users often get surprised because some comments seem to be missing under a post. This happens most often because comments dont have any language specified and are tagged as "Undetermined" language. So if that language isnt enabled in settings, lots of comments appear to be missing.

### Describe the solution you'd like

To make users aware of this, it would be useful to show messages like the following:

- 2 comments hidden in language Undetermined which is not enabled in your settings
- 1 comment hidden in language French which is not enabled in your settings

Next to these messages there could be a button "temporarily show all messages". Additionally, clicking individual messages might enable the respective language. Helping users to view comments in "undetermined" language is clearly the priority here.

Similar messages could also be added to indicate hidden comments from bot accounts or blocked users.

**Comment by aschrijver** [comment]
I would mention that the Languages listbox in Settings would be more intuitive if it was a list of checkbox items. Now the way to select multiple languages is by holding the `Ctrl`-key and creating the multi-select, which is awkward and regular users may not readily think of.

**Comment by Nutomic** [comment]
With Lemmy 0.17.3 undetermined language is automatically always enabled.

https://github.com/LemmyNet/lemmy/pull/2851

**Comment by dessalines** [comment]
I don't think this can be done in the front end. Hidden / wrong language comments are never returned at all from the API.
